.github/pull_request_template.md

```diff
@@ -0,0 +1,29 @@
+<!-- The following is intended to be helpful to you. Feel free to remove anything that is not. -->
+
+# JIRA Ticket: 
+MNTOR-
+
+
+<!-- When adding a new feature: -->
+
+# Description
+
+
+
+# Screenshot (if applicable)
+
+Not applicable.
+
+# How to test
+
+
+
+# Checklist (Definition of Done)
+- [ ] l10n changes have been submitted to the l10n repository, if any.
```

Comment:
For Relay, they have a separate repo/server I believe, and this line is more applicable to them.  For us, we might say something like below?
```suggestion
- [ ] Localization strings (if needed) have been added.
```